src/main/kotlin/no/nav/syfo/kafka/varselbus/Util.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.kafka.varselbus
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import no.nav.syfo.kafka.varselbus.domain.DineSykmeldteHendelse
+import no.nav.syfo.kafka.varselbus.domain.OppfolgingsplanNLVarselData
+import no.nav.syfo.kafka.varselbus.domain.EsyfovarselHendelse
+import no.nav.syfo.kafka.varselbus.domain.HendelseType
+import no.nav.syfo.kafka.varselbus.domain.HendelseType.*
+import no.nav.syfo.kafka.varselbus.domain.DineSykmeldteHendelse.*
+import org.apache.commons.cli.MissingArgumentException
+import java.io.IOException
+
+const val NL_OPPFOLGINGSPLAN_SENDT_TIL_GODKJENNING_TEKST = "1 oppfølgingsplan til godkjenning"
```

Comment:
Disse tekstene, som etterhvert kommer til å bli veldig mange, fortjener å ligge i en egen fil

Reply:
Dette er vel også teksten som skal benyttes til Dine sykmeldte. Man kan kanskje se for seg at teksten for det samme varselet som skal sendes til arbeidsgivernotifikasjoner har en annen tekst. Kanskje bør man ha med `DINE_SYKMELDTE` på noen måte i navnet?

Reply:
Kan bytte ut NL med DINE_SYKMELDTE

Reply:
Lagt i egen fil (VarselTexts) i no.nav.syfo